packages/lesswrong/components/tagging/EAAllTagsPage.tsx

```diff
@@ -12,6 +12,11 @@ import { taggingNameCapitalSetting, taggingNamePluralCapitalSetting, taggingName
 import { tagCreateUrl, tagUserHasSufficientKarma } from '../../lib/collections/tags/helpers';
 
 const styles = (theme: ThemeType): JssStyles => ({
+  coreTagsTitle: {
+    [theme.breakpoints.down('sm')]: {
+      marginTop: 20
```

Comment:
What's this for? I'm kinda confused why it doesn't need to be `EA_FORUM_HEADER_HEIGHT`

Reply:
I **think** this was a problem before I updated the site header - I remember noticing it at some point after we added the "Core Topics" list

Reply:
But why don't we need to use the height here?

Reply:
Oh I see what you mean, yeah it prob does depend on the height - I'll do that in some other PR